wrapper.js

```diff
@@ -99,13 +99,18 @@ function setupMethods (soljson) {
         } else {
           response.pipe(mem);
           response.on('end', function () {
-            cb(null, setupMethods(requireFromString(mem.toString())));
+            var solcContent = mem.toString();
+            cb(null, setupMethods(requireFromString(solcContent)), solcContent);
           });
         }
       }).on('error', function (error) {
         cb(error);
       });
     },
+    // Loads the compiler from a given file path. 
+    loadVersionFromFilePath: function (filePath) {
```

Comment:
This should probably be in line with the above method (`loadRemoteVersion`) and called `loadLocalVersion`.